modules/microshift-config-yaml.adoc

```diff
@@ -44,21 +49,22 @@ debugging:
 <8> Subject Alternative Names for API server certificates.
 <9> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
 
+[id="microshift-yaml-custom_{context}"]
+== Custom settings
+To create custom configurations, you must create a `config.yaml` file in the `/etc/microshift/config.yaml` directory, then change any settings that should override the defaults before starting or restarting {product-title}.
+
 [IMPORTANT]
 ====
 Restart {product-title} after changing any configuration settings to have them take effect. {product-title} reads the configuration file only on start.
```

Comment:
A little outside of scope, but could you be more specific of `on start`. Is it on start of MicroShift ? I read that and think start of the system.